CHANGELOG.md

```diff
@@ -2,19 +2,14 @@
 
 ## Unreleased
 * Start & Stop Navigation in existing Map
-    - Removed: `NavigationViewController(for route: Route, dayStyle: Style, routeController: RouteController? = nil, locationManager: NavigationLocationManager? = nil, voiceController: RouteVoiceController? = nil)` use `NavigationViewController(dayStyleURL: URL, nightStyleURL: URL? = nil,directions: Directions = .shared, voiceController: RouteVoiceController = RouteVoiceController())` followed by `startNavigation(with route: Route)` instead.
-    - To simulate a route, pass a `SimulatedLocationManager` to `startNavigation()` function: 
+    - Removed: `NavigationViewController(for route: Route, dayStyle: Style, routeController: RouteController? = nil, locationManager: NavigationLocationManager? = nil, voiceController: RouteVoiceController? = nil)` use `NavigationViewController(dayStyleURL: URL, nightStyleURL: URL? = nil,directions: Directions = .shared, voiceController: RouteVoiceController = RouteVoiceController())` followed by `startNavigation(with route: Route, animated: Bool)` instead.
+    - To simulate a route, pass a `SimulatedLocationManager` to `startNavigation()` function:
 
     ```swift
     let vc = NavigationViewController(dayStyleURL: AppConfig().tileserverStyleUrl)
-
-    if Env.current.simulateLocationForTesting {
-        let simulatedLocationManager = SimulatedLocationManager(route: route)
-        simulatedLocationManager.speedMultiplier = 5
-        vc.startNavigation(with: route, locationManager: simulatedLocationManager)
-    } else {
-        vc.startNavigation(with: route)
-    }
+    let simulatedLocationManager = SimulatedLocationManager(route: route)
+    simulatedLocationManager.speedMultiplier = 5
+    vc.startNavigation(with: route, animated: false, locationManager: simulatedLocationManager)
```

Comment:
Why did you remove this? This way navigation will always simulate the route, which is helpful while working in the simulator but the opposite of what users want. It's not intuitive to omit this parameter (when the changelog says to pass it) to get default behaviour back. Maybe a comment here would be better suited

Reply:
I thought this was only supposed to be an example for the immediately preceding bullet point, which was about simulating location. My mistake! I'll revert it.